app/src/org/commcare/network/HttpRequestEndpointsMock.java

```diff
@@ -31,7 +31,13 @@ public static void setCaseFetchResponseCodes(Integer[] responseCodes) {
 
     @Override
     public HttpResponse makeCaseFetchRequest(String baseUri, boolean includeStateFlags) throws ClientProtocolException, IOException {
```

Comment:
@phillipm I changed the implementation of this method to default to returning a response with result code 200 if `catchFetchResponseCodeStack` is empty, so that I could continue using this in the way I had been before without needing to call `setCaseFetchResponseCodes`, which feels wrong to do in live code. Let me know if you see any issue with this resolution.